src/apps/loan-list/modal/renew-loans-modal-content.tsx

```diff
@@ -131,6 +139,24 @@ const RenewLoansModalContent: FC<RenewLoansModalContentProps> = ({
             );
           })}
         </ul>
+        {!topButtonsVisible && (
+          <div
+            className="modal-loan__buttons modal-loan__buttons--bottom"
+            style={{ position: "fixed", left: 0, bottom: 0, right: 0 }}
+          >
+            <CheckBox
+              onChecked={selectAll}
+              id="checkbox-select-all"
+              label={checkboxBottomLabel}
+            />
+            <button
+              type="button"
+              className="btn-primary btn-filled btn-small arrow__hover--right-small"
+            >
+              {buttonBottomLabel}({renewable})
+            </button>
```

Comment:
Consider if this could use our `Button` component.

Reply:
Definitely, will fix